documentation/modules/overview/con-configuration-points-listeners.adoc

```diff
@@ -3,42 +3,39 @@
 // assembly-deploy-client-access.adoc
 
 [id="configuration-points-listeners-{context}"]
-= Configuring listeners to connect to Kafka brokers
+= Configuring listeners to connect to Kafka
 
 [role="_abstract"]
-Use listeners for client connection to Kafka brokers.
+Use listeners to enable client connections to Kafka.
 Strimzi provides a generic `GenericKafkaListener` schema with properties to configure listeners through the `Kafka` resource.
-The `GenericKafkaListener` provides a flexible approach to listener configuration.
-You can specify properties to configure _internal_ listeners for connecting within the Kubernetes cluster or _external_ listeners for connecting outside the Kubernetes cluster.
 
-Specify a connection `type` to expose Kafka in the listener configuration.
-The type chosen depends on your requirements, and your environment and infrastructure.
-The following listener types are supported:
+When configuring a Kafka cluster, you specify a listener `type` based on your requirements, environment, and infrastructure. 
+Services, routes, load balancers, and ingresses for clients to connect to a cluster are created according to the listener type.
 
-Internal listeners::
+Internal and external listener types are supported.
+
+Internal listeners:: Use internal listener types to connect clients within a kubernetes cluster.
 +
 * `internal` to connect within the same Kubernetes cluster
 * `cluster-ip` to expose Kafka using per-broker `ClusterIP` services
-External listeners::
++
+Internal listeners use a headless service and the DNS names assigned to the broker pods. 
+By default, they use the Kubernetes service DNS domain (typically `.cluster.local`). 
+However, you can customize this configuration using the `useServiceDnsDomain` property. 
```

Comment:
They do not use the DNS suffix by default IIRC. `useServiceDnsDomain` defaults to false.

Reply:
Right. According to the API ref, they don't use the DNS suffix by default.